topmed_automation.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+PM_CODE=$1
+PM_PATH=$2
+
+SUB_ROOT=/groups/submissions/metadata/v1/topmed
+SUB_PATH=${PM_PATH:56}
+
+ASP_ROOT=/aspera/share/globusupload/submissions
+
+#########
+# Usage #
+#########
+usage() {
+   err "usage: $(basename "$0") PM_CODE PM_PATH"
+   err "    Run TOPmed automation for validation and copy steps."
+   err "    PM_CODE: code provided by the project manager"
+   err "    PM_PATH: path provided by the project manager"
+   err
+   exit 1
+} >&2
+
+err() { echo "$@" >&2; }
+
+if [[ $# -ne 2 ]]; then
+   usage
+fi
+
+###############
+# Function(s) #
+###############
+tmux_session_exist () {
+    if [ $? != 0 ]; then
+        echo "ERROR: tmux session already exist."
+        echo "Make sure to exit previous sessions before continuing"
+        exit 1
+    fi
+}
+
+
+#########
+# Setup #
+#########
+
+# TODO: Aspera space issue.
+# Remind user?
+
+# Check if shepherd config file exist
+
+CONFIG_FILE=/users/$USER/.config/shepherd.yaml
+
+if [ ! -f $CONFIG_FILE ]; then
+    cat << EOF > $CONFIG_FILE
+sub_root: $SUB_ROOT
+asp_root: $ASP_ROOT
+EOF
+fi
+
+# Check if tmux sessions exist
+# tmux new -s topmed-copy -d
+# tmux_session_exist
+tmux new -s topmed-login -d
```

Comment:
Note that if this command creates a new session, then the window will have a size of 80x24, unless the command includes values for `-x` and `-y`. This could affect line wrapping of output from commands running inside tmux (if those commands check the window size). An example is the `ls` command which wraps output based on terminal size.